CI: golangci-lint flaky?

```
[6](https://github.com/podman-container-tools/podman/actions/runs/27210239644/job/80336972777#step:8:37)
VERSION=2.12.2 ./hack/install_golangci.sh
golangci-lint has version 2.12.2 built with go1.26.2 from c0d3ddc9 on 2026-05-06T11:07:58Z
Using existing /home/ubuntu/golangci-lint-2.12.2-linux-amd64/golangci-lint
hack/golangci-lint.sh
Linting for GOOS=linux
+ /home/ubuntu/golangci-lint-2.12.2-linux-amd64/golangci-lint run --build-tags=apparmor,seccomp,selinux
0 issues.
+ /home/ubuntu/golangci-lint-2.12.2-linux-amd64/golangci-lint run --build-tags=apparmor,seccomp,selinux,systemd
0 issues.
+ /home/ubuntu/golangci-lint-2.12.2-linux-amd64/golangci-lint run --build-tags=apparmor,seccomp,selinux,remote
0 issues.
CGO_ENABLED=0 GOOS=windows hack/golangci-lint.sh
Linting for GOOS=windows
+ /home/ubuntu/golangci-lint-2.12.2-linux-amd64/golangci-lint run --build-tags=remote,containers_image_openpgp
Error: cmd/podman/containers/ps.go:389:3: directive `//nolint:staticcheck` is unused for linter "staticcheck" (nolintlint)
		//nolint:staticcheck
		^
1 issues:
* nolintlint: 1
make: *** [Makefile:294: golangci-lint] Error 1
Error: Process completed with exit code 2.
```

https://github.com/podman-container-tools/podman/actions/runs/27210239644/job/80336972777

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -99,6 +99,7 @@ jobs:
         with:
           version: ${{ steps.gv.outputs.version }}
           install-only: true
+          skip-cache: true # cache causes flaky results https://github.com/podman-container-tools/podman/issues/28893
 
       - name: Install pre-commit
         run: pipx install pre-commit

